roles/backpack/tasks/main.yml

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
Could you add to the benchmark state: `metadata_collection` ?

Reply:
updated to include blocking states.